sdk/lib/_http/websocket_impl.dart

```diff
@@ -1022,7 +1022,9 @@ class _WebSocketImpl extends Stream with _ServiceObject implements WebSocket {
         path: uri.path,
         query: uri.query,
         fragment: uri.fragment);
-    return _httpClient.openUrl("GET", uri).then((request) {
+        // Use Custom HTTP Client first is it exists, else
```

Comment:
I think this comment can be omitted as it just describes what is happening on the next line.
Ideally comments are answering the question "why?", rather than "what?", because normally code answers "what?" question already.